### PR TITLE
Use Endless Key loading screen

### PR DESCRIPTION
**Pull request by dylanmccall**
Closes: https://github.com/endlessm/endless-key-flatpak/issues/68

### CODE REVIEW

File: build-aux/flatpak/modules/python3-kolibri-explore-plugin.json [hunk 1]
```diff
@@ -14,9 +15,16 @@
         },
         {
             "type": "archive",
-            "url": "https://github.com/endlessm/kolibri-explore-plugin/releases/download/v6.47.0/apps-bundle.zip",
-            "sha256": "25e758ebf7e1e6c8df270fbf87268128dce6e2c3d2c6477b17b77d4332611a6a",
+            "url": "https://github.com/endlessm/kolibri-explore-plugin/releases/download/v7.0.0rc3/apps-bundle.zip",
```
Comment (dbnicholson) [inline build-aux/flatpak/modules/python3-kolibri-explore-plugin.json:18]: I think you also have to the wheel to 7.0.0rc3, right? Or if this is purely about replacing the pre-kolibri loading screen, then I would leave the apps bundle on the old release. I have my doubts that the old frontend and the new template-ui will play nice together.

**Review by dbnicholson — CHANGES_REQUESTED** [review]
This looks great! Just one minor question about keeping the versions in sync.

File: build-aux/flatpak/modules/python3-kolibri-explore-plugin.json (hunk 1 above)
Comment (starnight) [reply build-aux/flatpak/modules/python3-kolibri-explore-plugin.json:18]: I built this branch and tested locally.  It shows the Endless Key loading animation, then downloads Explorer content pack automatically.
Screenshot from 2023-09-20 16-03-51
Screenshot from 2023-09-20 16-06-41

If this PR does only update the loading animation, then it works.  But, ya, try to double confirm **only** the loading screen, no wheel (7.0.0rc3)?

Comment (dylanmccall) [reply build-aux/flatpak/modules/python3-kolibri-explore-plugin.json:18]: Oh, this is still using v6.47.0. I meant to change it to v7.0.0rc3, but forgot to update the plugin itself :) It should be working as expected with the newest commit.

**Review by dbnicholson — APPROVED** [review]
Looks good. Please rebase and merge.

**Pull request merged**